dv/parameters/KDV01_CubeOrangePlusDefault.param

```diff
@@ -28,55 +28,55 @@ AHRS_COMP_BETA,0.1
 AHRS_EKF_TYPE,3.0
 AHRS_GPS_GAIN,1.0
 AHRS_GPS_MINSATS,6.0
-AHRS_GPS_USE,1.0
+AHRS_GPS_USE,2.0
```

Comment:
This controls whether to use dead-reckoning or GPS based navigation. If set to 0 then the GPS won't be used for navigation, and only dead reckoning will be used. A value of zero should never be used for normal flight. Currently this affects only the DCM-based AHRS: the EKF uses GPS according to its own parameters. A value of 2 means to use GPS for height as well as position - both in DCM estimation and when determining altitude-above-home.